src/test/java/org/mockitoutil/JUnitResultAssert.java

```diff
@@ -47,6 +50,17 @@ public JUnitResultAssert fails(int expectedFailureCount, Class expectedException
     }
 
     /**
+     * Expects single failure with specific exception and exception message.
+     * Automatically filters line numbers from exception messages.
+     */
+    public JUnitResultAssert fails(Class expectedException, String exceptionMessage) {
+        fails(1, expectedException);
+        Failure f = result.getFailures().iterator().next();
```

Comment:
Interesting. iterator().next() is a fairly common idiom to get first element, for example from a set. It's a good idea to put it in Iterables. There is no harm to call hasNext so I'll do it. Thanks for the suggestion!